src/test/java/com/example/amazoncom/BrowserStackSetup.java

```diff
@@ -0,0 +1,42 @@
+package com.example.amazoncom;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class BrowserStackSetup {
+  String driverPath = "C:\\geckodriver.exe";
+  WebDriver driver;
+  BrowserStackHomePage objBrowserStackHomePage;
+  BrowserStackSignUpPage objBrowserStackSignUpPage;
+  @BeforeTest
+  public void setup() {
+    System.setProperty("webdriver.chrome.driver",
```

Comment:
Do not use an absolute path. use the path from the project root.
and pls use webdriver manager dependency

Reply:
you had the correct implementation in your removed amazon test class:
`WebDriverManager.chromedriver().setup();`